src/cookies.test.js

```diff
@@ -1,60 +1,59 @@
-import { createCookie, getCookie } from './cookies'
+/* eslint-disable no-undef */
+import { createCookie, getCookie } from './cookies';
 
 /* https://stackoverflow.com/questions/179355/clearing-all-cookies-with-javascript */
 function deleteAllCookies() {
-    var cookies = document.cookie.split(";");
-
-    for (var i = 0; i < cookies.length; i++) {
-        var cookie = cookies[i];
-        var eqPos = cookie.indexOf("=");
-        var name = eqPos > -1 ? cookie.substr(0, eqPos) : cookie;
-        document.cookie = name + "=;expires=Thu, 01 Jan 1970 00:00:00 GMT";
-    }
+  const cookies = document.cookie.split(';');
+  for (let i = 0; i < cookies.length; i++) {
+    const cookie = cookies[i];
+    const eqPos = cookie.indexOf('=');
+    const name = eqPos > -1 ? cookie.substr(0, eqPos) : cookie;
+    // eslint-disable-next-line prefer-template
+    document.cookie = name + '=;expires=Thu, 01 Jan 1970 00:00:00 GMT';
```

Comment:
Use template instead of disabling the rule
document.cookie = `${name}=;expires=Thu, 01 Jan 1970 00:00:00 GMT`